pkg/kapis/devops/v1alpha2/handler.go

```diff
@@ -34,6 +40,17 @@ type PipelineSonarHandler struct {
 	pipelineSonarGetter devops.PipelineSonarGetter
 }
 
+func jenkinsProxy(request *restful.Request, response *restful.Response) {
+	u := request.Request.URL
+	u.Host = "allinone.ks.dev.chenshaowen.com:30180"
```

Comment:
What's this? why the host has to be this ?

Reply:
It's testing code. Removed.